client/rpcserver/rpcserver_test.go

```diff
@@ -88,9 +88,8 @@ func (r *TReader) Close() error { return nil }
 
 type TConn struct {
 	msg       []byte
-	reads     [][]byte      // data for ReadMessage
-	respReady chan []byte   // signal from WriteMessage
-	close     chan struct{} // Close tells ReadMessage to return with error
```

Comment:
`close` is actually used now.  Pushed a fix that uses it in Close: https://github.com/decred/dcrdex/commit/5a6faa1847121036650c498dcbbbbd1cb85a5b56